pkg/bundle/sync_test.go

```diff
@@ -1729,3 +1729,22 @@ func Test_certAlias(t *testing.T) {
 		t.Fatalf("expected alias to be %q but got %q", expectedAlias, alias)
 	}
 }
+
+func TestBundlesDeduplication(t *testing.T) {
+	// list of certificates
+	bundle := []string{
+		dummy.TestCertificate1,
+		dummy.JoinCerts(dummy.TestCertificate1, dummy.TestCertificate3),
+	}
+
+	// test bundle
+	testBundle := []string{
+		dummy.TestCertificate1,
+		dummy.TestCertificate3,
+	}
+
+	resultBundle := deduplicateBundles(bundle)
```

Comment:
Well done finding the issue with bundles containing multiple certificates. Do you mind adding a few more test cases to verify everything works as expected?

Reply:
Sure, I will do that.